core-field-editors/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@contentful/field-editors",
+  "version": "0.0.1",
+  "description": "",
+  "main": "lib/all.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint bin/bundle src",
+    "prepublish": "babel src --out-dir lib",
+    "watch": "babel src --out-dir lib --watch"
+  },
+  "author": "Thomas Scholtes <thomas@contentful.com>",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-cli": "^6.11.4",
+    "babel-core": "^6.13.2",
+    "babel-preset-es2015": "^6.13.2",
+    "babelify": "^7.3.0",
+    "browserify": "^13.1.0",
+    "eslint": "^3.4.0",
+    "eslint-config-standard": "^6.0.0",
+    "eslint-plugin-promise": "^2.0.1",
+    "eslint-plugin-standard": "^2.0.0",
+    "uglifyify": "^3.0.2",
+    "vdom-to-html": "^2.3.0",
+    "yargs": "^5.0.0"
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ]
+  },
+  "dependencies": {
```

Comment:
I think it is meaningful. If you are using the package as a library you are only going to use the transpiled files in `./lib`. And these only require the production dependencies. If you want to bundle an editor you are not going to use the NPM package but rather the repo with `npm install`.